package.json

```diff
@@ -280,21 +289,26 @@
 		"prepare": "tsc -p ./",
 		"compile": "tsc -p ./",
 		"watch": "tsc -w -p ./",
-		"test": "mocha -u tdd ./out/test/"
+		"test": "mocha -u tdd ./out/test/",
+		"lint": "eslint -c .eslintrc.js --ext .ts ./"
 	},
 	"devDependencies": {
-		"@types/mocha": "^5.2.7",
-		"@types/node": "^13.13.0",
-		"@types/vscode": "^1.44.0",
+		"@types/mocha": "^10.0.1",
+		"@types/node": "^20.6.3",
+		"@types/vscode": "^1.82.0",
```

Comment:
see above: is there a _need_ for a newer type? If yes, then do we really need to depend on the most up-to-date one?